src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,98 @@
+import { Link, NavLink } from "react-router-dom";
+
+const navbar = () => {
+  return (
+    <div className="navbar flex flex-row justify-between items-center bg-black w-screen h-24 px-24">
+      <div className="container_pages">
+        {/* logo & home link */}
+        <Link
+          to="/"
+          className="
+           text-white
+            cursor-pointer
+            text-2xl
+            -tracking-wider 
+            leading-7 
+            font-noto 
+            font-bold
+            mr-20"
+        >
+          Stellar
+        </Link>
+
+        {/* pages links */}
+        <NavLink
+          to="/planatrip"
+          className={({ isActive }) => {
+            return (
+              "text-white cursor-pointer text-xs leading-7 font-robo font-bold hover:text-red-700 transition-colors duration-200 delay-75 mr-10" +
+              (!isActive ? "" : " text-red-700")
+            );
+          }}
+        >
+          Plan a trip
+        </NavLink>
+        <NavLink
+          to="/discover"
+          className={({ isActive }) => {
+            return (
+              "text-white cursor-pointer text-xs leading-7 font-robo font-bold hover:text-red-700 transition-colors duration-200 delay-75 mr-10" +
+              (!isActive ? "" : " text-red-700")
+            );
+          }}
+        >
+          Discover
+        </NavLink>
+        <NavLink
+          to="/checkflights"
+          className={({ isActive }) => {
+            return (
+              "text-white cursor-pointer text-xs leading-7 font-robo font-bold hover:text-red-700 transition-colors duration-200 delay-75 mr-10" +
+              (!isActive ? "" : " text-red-700")
+            );
+          }}
+        >
+          Check flights
+        </NavLink>
+      </div>
+
+      {/* login and register links */}
+      <div className="container_access">
```

Comment:
Nit / curiosity: Is this div with a custom classname needed? Why or why not? 

I'm highlighting this to make sure you're thinking about when & why you're adjusting the structure of the DOM. If you're creating more structure, or applying custom class names, make sure you know why and have a good reason for it - is there a better way? Is it absolutely necessary? Does it make the layout more complex or easier to maintain?